doc-content/enterprise-only/openshift/operator-deploy-kieserver-proc.adoc

```diff
@@ -26,23 +26,23 @@ Optionally, you can set custom configuration for {KIE_SERVERS}. In this case, de
+.. In the *KIE Server container deployment* field, enter the identifying information of the services (KJAR files) that the deployment must take from the result of a Source to Image (S2I) build. The format is `<containerId>=<groupId>:<artifactId>:<version>` or, if you want to specify an alias name for the container, `<containerId>(<aliasId>)=<groupId>:<artifactId>:<version>`. You can provide two or more KJAR files using the `|` separator, as illustrated in the following example: `containerId=groupId:artifactId:version|c2(alias2)=g2:a2:v2`.
 .. If your OpenShift environment does not have a connection to the public Internet, enter the URL of the Maven mirror that you set up according to <<offline-repo-proc_{context}>> in the *Maven mirror URL* field.
 .. If you want to use an S2I build, the *Artifact directory* field, enter the path within the project that contains the required binary files (KJAR files and any other necessary files) after a successful Maven build. Normally this directory is the target directory of the build. However, you can provide prebuilt binaries in this directory in the Git repository.
 .. If you want to use a custom base {KIE_SERVER} image for the S2I build, click *Set Base build image* and then enter the name of the image stream in the *Name* field. If the image stream is not in the `openshift` namespace, enter the namespace in the *Namespace* field. If you want to use a Docker image name and not an OpenShift image stream tag, change the *Kind* value to `DockerImage`.
-.. If you want to create the immutable server from source in a Git repository using the Source to Image (S2I) process, click the *Set Git source* button and enter information in the following fields:
+.. Click the *Set Git source* button and enter information in the following fields:
 *** *S2I Git URI*:The URI for the Git repository that contains the source for your services.
 *** *Reference*: The branch in the Git repository.
 *** *Context directory*: (Optional) The path to the source within the project downloaded from the Git repository. By default, the root directory of the downloaded project is the source directory.
-+
-[NOTE]
-====
-If you do not configure a Git source, the immutable {KIE_SERVER} does not use an S2I build. Instead, it pulls the artifacts that you define in the *KIE Server container deployment* field from the configured Maven repository.
-====
-+
-.. If you are using S2I and want to set a Git Webhook so that changes in the Git repository cause an automatic rebuild of the {KIE_SERVER}, click *Add new Webhook*. Then select the type of the Webhook in the *Type* field and enter the secret string for the Webhook in the *Secret* field.
+.. If you are  want to set a Git Webhook so that changes in the Git repository cause an automatic rebuild of the {KIE_SERVER}, click *Add new Webhook*. Then select the type of the Webhook in the *Type* field and enter the secret string for the Webhook in the *Secret* field.
```

Comment:
Suggestion - correct typo, remove then and that, and adopt UI description for the `Type` field:

`s/If you are  want to set/ If you want to set`

`s/so that changes/so changes`

`s/Then select the/Select the`

Question - is the Type field a drop down? if so:

`s/Select the type of the Webhook in the *Type* field and/Select the Webhook type from the *Type* list and`